src/main/kotlin/hu/kirdev/schpincer/web/ConfigureController.kt

```diff
@@ -472,6 +479,18 @@ class ConfigureController {
                 table.addCell(cell)
             }
         }
+        return ordersList.size;
+    }
+
+    private fun addEmptyRows(table: PdfPTable, export: ExportType, emptyRows: Int) {
+        for (i in 1..emptyRows) {
+            export.fields.forEach { _ ->
+                val cell = PdfPCell()
+                cell.phrase = Phrase(" ") // <-- Needs a whitespace
```

Comment:
Why does it need a white space? I mean I have some tips, but if you make a comment make it clear for anyone.
For example:
```
// NOTE: Empty phrases are ignored. Therefore at least a whitespace character is needed to be added.
```